### ISSUE TITLE
connection.autocommit(False) not working as of 0.2.1

### ISSUE THREAD

**Issue by zzzeek** [long2ice/asyncmy#21]
the following test case succeeds in 0.2.0, fails in 0.2.1, 0.2.2:

```py
import asyncio

import asyncmy

async def main():

    conn = await asyncmy.connect(user="scott", password="tiger", database="test", host="localhost")

    await conn.autocommit(True)

    cursor = conn.cursor()

    await cursor.execute("SELECT @@autocommit;")

    assert await cursor.fetchone() == (1, )

    await cursor.close()

    # turn autocommit off
    await conn.autocommit(False)

    cursor = conn.cursor()

    await cursor.execute("SELECT @@autocommit;")

    assert await cursor.fetchone() == (0, )

    await cursor.close()

asyncio.run(main())
```

the second assertion fails:

```
$ python test3.py 
Traceback (most recent call last):
  File "/home/classic/dev/sqlalchemy/test3.py", line 30, in <module>
    asyncio.run(main())
  File "/usr/lib64/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib64/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/classic/dev/sqlalchemy/test3.py", line 26, in main
    assert await cursor.fetchone() == (0, )
AssertionError

```

**Comment by long2ice** [comment]
Could you test again?

**Comment by long2ice** [comment]
released v0.2.3
